app/models/work.rb

```diff
@@ -14,6 +14,19 @@ class Work < ApplicationRecord
 
   has_one :region, through: :location
 
+  def authors
+    # TODO is there a better way to get this information?
+    # I am having trouble when starting from "self.people" type relationship
+    all_people = Person
+      .joins(work_roles: :role)
+      .where(work_roles: { work_id: id })
+
+    # now filter those people to those who are poets / authors
+    all_people.where(roles: { name: "Poet" })
+      .or(all_people.where(roles: { name: "Author" }))
```

Comment:
I think you had earlier done a similar query with code like `roles: { name: ["Author", "Poet"] })`. Are the results of that formatting different than using the `.or` here?

Reply:
Oh yeah, I think you're totally right that I can use the nicer syntax I figured out later.  That will make things easier!